signatures/golang/anti_debugging_ptraceme.go

```diff
@@ -25,6 +25,7 @@ func (sig *AntiDebuggingPtraceme) GetMetadata() (detect.SignatureMetadata, error
 		ID:          "TRC-102",
 		Version:     "1",
 		Name:        "Anti-Debugging detected",
+		EventName:   "anti_debugging_ptraceme",
```

Comment:
all these simple EventName changes should be on its own commit (to clean this up a bit).

Reply:
@rafaeldtinoco Yes, just want to be sure everyone agreed with the new UX, as a whole PR before breaking it on refactoring without a context. The update of types+event_name can be separated on a different PR, will do it in now.